pkg/strategy/xgap/strategy.go

```diff
@@ -257,15 +254,69 @@ func (s *Strategy) makeSpread(ctx context.Context, bestBid, bestAsk types.PriceV
 	return err
 }
 
+func adjustPositionOrder(symbol string, tradingMarket types.Market, currentPosition *types.Position, bestBid, bestAsk types.PriceVolume) (orderForms []types.SubmitOrder) {
+	if currentPosition == nil {
```

Comment:
should check this outside of this function?

we should assume currentPosition is not nil for this function

Reply:
ok

Reply:
Done.